newlib/libc/include/complex.h

```diff
@@ -14,6 +14,7 @@
 
 #include <sys/cdefs.h>
 
+#ifdef HAVE_COMPLEX
```

Comment:
We shouldn't need changes in this file, but we should change the build system to not install it.